gitless/core.py

```diff
@@ -131,7 +131,7 @@ def merge_base(self, b1, b2):
 
   def _fuse_commits_fp(self, b):
     return os.path.join(
-        self.path, 'GL_FUSE_CIS_{0}'.format(b.branch_name.replace('/', '_')))
+        self.path, 'GL_FUSE_CIS_{0}'.format(b.branch_name.replace('/', '_'))) # TODO is this safe?
```

Comment:
I believe it is safe. These are just files that we save in the .git directory to keep track of stuff. There might be one of these per branch (which is why we append to the file name the name of the branch). The '/' for '_'  replacement is just to prevent having files that have a slash in their names. So, for example, instead of creating a file with name "GL_FUSE_CIS_features/foo-bar" we would create one with name "GL_FUSE_CIS_features_foo-bar" instead.